index.html

```diff
@@ -35,15 +36,15 @@ <h2 id="citationHeadingRef" class="citation-heading"><b>" Lorem Ipsum is simply
                             "</b></h2>
                 </div>
 
-                <button id="coursesButtonRef" class="courses-button home-button">
+                <button id="coursesButtonRef" class="courses-button home-button" slot="home-grid-button">
                     <h3>CORSI</h3> <img class="button-icons" src="images/courses.svg">
```

Comment:
Con astrai i botton i intendevo astrai tutta questa porzione di codice e cambia il ccontenuto in base alle esigenze dopo quando li aggiungi. In questo modo hai astratto solamente lo style. Riprova